backend/azureblob/azureblob.go

```diff
@@ -1199,8 +1219,28 @@ func (o *Object) Update(in io.Reader, src fs.ObjectInfo, options ...fs.OpenOptio
 	if err != nil {
 		return err
 	}
+	// Refresh metadata on object
 	o.clearMetaData()
-	return o.readMetaData()
+	err = o.readMetaData()
+	if err != nil {
+		return err
+	}
+
+	// Now, set blob tier based on configured access tier
```

Comment:
Presumably we only need to actually set the blob tier if it differs from the default?  If so then we can save an HTTP roundtrip for the default case.